app/policies/application_policy.rb

```diff
@@ -174,10 +174,6 @@ def edit?
     update?
   end
 
-  def manage?
```

Comment:
Reading note: I suspect that `manage?` is only relevant for articles as they are the only ones that are published.  @lightalloy: are articles the only "published" thing?  Wondering about podcasts in particular.

I think removing this may be risky, but you're more aware of the specific details at this point.

Reply:
There are several models that have `published`, but `Policy#manage?` is not used for them. As for the podcasts, `PodcastPolicy` doesn't exist, for publishing we use `update` and `InternalPolicy#access?` to authorize.
As I see, other models that have `published` don't use `Policy#manage?` and use more general authorization. `HtmlVariant` has its own policy but it seems to not use `manage?`.
Do you suspect there could be places where `manage?` is used implicitly?

Reply:
Excellent sleuthing. 

> Do you suspect there could be places where `manage?` is used implicitly?

I'm 95% comfortable that we don't have an implicit use; and after looking for `def manage` the only place I see that is in the ArticlesController, so the policy should work.

All told, let's go with your implementation.